src/files.js

```diff
@@ -632,6 +632,7 @@ FOAM_FILES([
   { name: "foam/comics/v2/DAOUpdateView" },
   { name: "foam/comics/v2/NamedViews" },
   { name: "foam/comics/v2/ViewSpecWithJava" },
+  { name: "foam/comics/v2/DataSummaryCard" },
```

Comment:
Let's leave this out for now since it's not fully baked and not really being used yet (unless I'm missing something?)

Reply:
Removing